src/i18n/content/jp/docs/logs/ui-data/obfuscation-ui.mdx

```diff
@@ -4,7 +4,7 @@ metaDescription: How to hash or mask sensitive data in logs you send to New Reli
 translationType: machine
 ---
 
-import obfuscationCreateExpressionUi from 'images/obfuscation-create-expression-ui.png'
+import obfuscationCreateExpression from 'images/ObfuscationCreateExpression.png'
```

Comment:
You shouldn't need to manually update the translated files like this, right?

Reply:
@paperclypse I couldn't get it to build until I fixed these import statements in JP and KN.